frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -180,11 +180,17 @@ export default class Signup extends Vue {
       ui.error(T.loginPasswordTooShort);
       return;
     }
-      const registerParameters = {
+    if (this.email || this.parentEmail == null) {
+      ui.error(T.fillEmailCorrectly);
+      return;
+    }
```

Comment:
```suggestion
    if (!this.isU13 && this.email === null) {
      ui.error(T.loginEmailMissing);
      return;
    } else if(this.isU13 && this.parentEmail === null) {
      ui.error(T.loginParentEmailMissing);
      return;
    }
```

here we only need to check for the corresponding email, not both emails. also, it's better to give out separate error messages for both cases so it's clear which email must be filled. also, we normally try to prefix the translation string with the rough section where it is used (all the other translation strings have `login` as prefix, so at the very least consistency with the rest of the file is desirable).

can there be a test for this?